fediplay/main.py

```diff
@@ -2,6 +2,7 @@
 
 from os import path
 import sys
+import argparse
```

Comment:
So, I'm sorry that I pulled this rug out from under you, but I had a branch in progress that changed fediplay to use the excellent [Click](http://click.pocoo.org/) instead.

I think the way I'd rather see this work on the command-line rather than adding a switch is something like

    $ fediplay stream cybre.space @alice @bob@bob.space @carol

And then if we do tags later we can do

    $ fediplay stream cybre.space @alice '#tag'

or something similar. In effect, the first argument is always the instance, and follow-on arguments are user or tag filters.